File: HW4/HotBeverage.java
package org.example.HW4;

/*
Класс горячих напитков (наследник класса Напитков)
 */
public class HotBeverage extends Beverage {
    private int temperature;

    public int getTemperature() {
        return temperature;
    }

    public int setTemperature() {
        return this.temperature;
    }
    // конструктор класса (у каждого класса дб конструктор)
    public HotBeverage(String name, int price, int volume, int temperature) {
        setName(name);
        setPrice(price);
        setVolume(volume);
        this.temperature = temperature;
    }
    // показать температуру напитка
    public void sayTemperature(){
        System.out.println("Температура напитка " + getName() + " равна " + getTemperature());
    }
}

File: HW7/HotBeverage.py
from Beverage import Beverage


class HotBeverage(Beverage):
    def __init__(self, name, price, volume, temperature):
        super().__init__(name, price, volume)
        self._temperature = temperature

    @property
    def temperature(self):
        return self._temperature

    # def sayTemperature(self):
    #     print(f"Температура напитка {Beverage.name} равна {self.temperature}")

    def drink_info(self):
        print(f'Название: {self.name} Стоимость: {self.price}. Объём: {self.volume}. Температура: {self.temperature}')